mlrun/api/utils/sqlite_migration.py

```diff
@@ -0,0 +1,66 @@
+import typing
+
+import sqlite3_to_mysql
+
+from mlrun import mlconf
+from mlrun.utils import logger
+
+from .mysql import MySQLUtil
+
+
+class SQLiteMigrationUtil(object):
+    def __init__(self):
+        self._mysql_dsn_data = MySQLUtil.get_mysql_dsn_data()
+        self._converter = self._create_converter()
```

Comment:
Doesn't matter too much but I would call it migrator (or migration util or something)